web-automation-agent/src/consts.ts

```diff
@@ -0,0 +1,29 @@
+export const UNIQUE_ID_ATTRIBUTE = 'gid';
+// WEB AUTOMATION
+// TODO
+export const WHITELIST_TAGS_WEB_AUTOMATION = [
+    'html', 'body', 'title',
+    'main', 'footer', 'header', 'nav', 'section', 'article',
+    'div', 'p',
+    'h1', 'h2', 'h3', 'h4', 'h5', 'h6',
+    'table', 'thead', 'tbody', 'tr', 'th', 'td',
+    'a', 'button',
+    'form', 'input', 'label', 'select', 'option', 'textarea',
+];
+
+export const WHITELIST_ATTRIBUTES_WEB_AUTOMATION = [
```

Comment:
It could make sense to keep `alt` or [aria](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes) attributes as those give context to some clickable elements.